observability/ml-observability.md

```diff
@@ -0,0 +1,44 @@
+# Observability in Machine Learning  
+
+Development process of software system with machine learning component is more complex
+than traditional software as such systems change in three directions: the code, the model and the data.
+We can distinguish 2 stages of such system lifespan: experimentation and production.
+Both stages require specific approach to observability that we will discuss below:
+
+## Model experimentation and tuning
+
+Experimentation is a process of finding suitable machine learning model and its parameters via training and evaluating such models with one or more data sets.
```

Comment:
Nit: language details

Experimentation is the process of finding a suitable machine learning model and its parameters via training and evaluating the model with one or more datasets.

When developing and tuning machine learning models, the data scientists are interested in observing and comparing selected performance metrics for different parameters. They also need a reliable way to reproduce a training process, such that a given dataset and given parameters produces the same models.

Reply:
Nice wording! Thank you!